### ISSUE TITLE
Crucible gets "stuck" on one type of item and won't accept others

### ISSUE THREAD

**Issue by McFrugal** [davqvist/Exoria#238]
For example, when producing Seed Oil, if you throw in one type of seed it's likely to refuse to melt any other type of seed.  This can happen with leaves too.

This can only be fixed by breaking the crucible and re-placing it.

**Comment by davqvist** [comment]
Yeah I heard about that one, just pump the Seed Oil in a Barrel, Tank or whatever so that the Crucible is empty and ready to accept other seeds
